pkg/reconciler/pipelinerun/resources/apply_test.go

```diff
@@ -2330,6 +2330,38 @@ func TestApplyTaskResults_MinimalExpression(t *testing.T) {
 					}}},
 			},
 		}},
+	}, {
+		name: "Test array result substitution on minimal variable substitution expression - matrix",
```

Comment:
can you also add a test case for array result indexing in a matrix?

Reply:
That already exists.

Reply:
my bad!